### PR TITLE
PostgreSQL check

### PR DESCRIPTION
**Pull request by KhromovNikita**

### CODE REVIEW

File: scripts/develop/dependence.py [hunk 1]
```diff
@@ -375,12 +379,11 @@ def check_7z():
   return dependence
 
 def get_mysql_path_to_bin(mysqlPath = ''):
-  if (host_platform == 'windows'):
+  if (host_platform != 'windows'):
```
Comment (trofim24) [inline scripts/develop/dependence.py:382]: Error?

File: scripts/develop/dependence.py [hunk 2]
```diff
@@ -490,6 +493,165 @@ def uninstall_mysqlserver():
 
   return  code
 
+def get_postrgre_path_to_bin(postrgrePath = ''):
+  if (host_platform == 'windows'):
+    if (postrgrePath == ''):
+      postrgrePath = os.environ['PROGRAMW6432'] + '\\PostgreSQL\\13\\'
+    #return '"' + postrgrePath + '\\bin\\psql"'
```
Comment (trofim24) [inline scripts/develop/dependence.py:500]: Delete comment

File: scripts/develop/dependence.py [hunk 3]
```diff
@@ -84,7 +84,10 @@ def check_dependencies():
   checksResult.append(check_gruntcli())
   if (host_platform == 'windows'):
     checksResult.append(check_buildTools())
-  checksResult.append(check_mysqlServer())
+  if (config.option("server-addons") == 'mysql'):
```
Comment (trofim24) [inline scripts/develop/dependence.py:87]: sql-type ?

File: scripts/develop/dependence.py [hunk 4]
```diff
@@ -97,14 +100,15 @@ def check_dependencies():
...
-    install_args += ['--mysql-path', unicode(checksResult.mysqlPath)]
     if (host_platform == 'windows'):
       code = libwindows.sudo(unicode(sys.executable), install_args)
     elif (host_platform == 'linux'):
       get_updates()
       base.cmd_in_dir('./scripts/develop/', 'python', install_args)
 
-  return check_MySQLConfig(checksResult.mysqlPath)
+  if (config.option("server-addons") == 'mysql'):
+    return dependence.check_MySQLConfig(checksResult.sqlPath)
```
Comment (trofim24) [inline scripts/develop/dependence.py:110]: "dependence." - mistake?

File: scripts/develop/dependence.py (hunk 1 above)
Comment (KhromovNikita) [reply scripts/develop/dependence.py:382]: Fixed

**Pull request merged**